addon/components/utils/address-form.ts

```diff
@@ -83,4 +125,50 @@ export default class extends Component<UtilsAddressFormArgs> {
       return isEmpty(get(this.args.address, addressAttr)) || shortAddress || invalidCountryCode || invalidZipcode;
     });
   }
+
+  private fillInAddress(place: any): void {
+    let address1: string = '';
+    let zipcode: string = '';
+    let city: string = '';
+
+    for (const component of place.address_components.reverse()) {
+      const componentType = component.types[0];
+
+      switch (componentType) {
+        case 'street_number': {
+          address1 = `${component.long_name} ${address1}`;
+          break;
+        }
+        case 'route': {
+          address1 += component.long_name;
+          break;
+        }
+        case 'postal_code': {
+          zipcode = `${component.long_name}${zipcode}`;
+          break;
+        }
+        case 'postal_code_suffix': {
+          zipcode = `${zipcode}-${component.long_name}`;
+          break;
+        }
+        case 'locality':
```

Comment:
also, this looks unused

Reply:
It is used, the `locality` and `postal_town` execute the same method